src/FetchRequest.js

```diff
@@ -3,7 +3,7 @@
  * Functional Component 
  */
 
-export const FetchRequest = (props) => {
+const FetchRequest = (props) => {
```

Comment:
It might be worth YOU reading up on code style conventions for JS (just kidding love you) As far as I'm aware, all components should be capitalised and functional components are still components. Let me know if you find anything to the contrary - just going off Google! Re. the fetchrequest name - noted.